src/frontend/mod.rs

```diff
@@ -146,6 +146,32 @@ impl<'a, B: Backend<'a>> Frontend<'a, B> {
         }
     }
 
+    fn convert_text(&mut self, mut text: Cow<'a, str>) {
+        // test if the text contains a latex command
+        lazy_static! {
+            // latex actually isn't really parsable, so this is only a very rough estimate of common commands
+            // Expect: \commandname<arg>[arg]{arg}{arg}...
+            static ref LATEX_COMMAND: Regex = Regex::new(r"\\[a-zA-Z]+(?:$|<.*?>|\[.*?\]|\{.*?\})").unwrap();
+        }
+        if let Some(m) = LATEX_COMMAND.find(&text) {
+            let (start, end, len) = (m.start(), m.end(), m.as_str().len());
+            if start != 0 {
+                let (l, r) = text.split_at(start);
+                text = r;
+                self.buffer.push_back(Event::Text(l));
+            }
+            if len != text.len() {
+                let (l, r) = text.split_at(end);
+                self.buffer.push_back(Event::Latex(l));
+                self.buffer.push_back(Event::Text(r));
```

Comment:
Wait, what happens when there is a second latex command in the same piece of text?

Reply:
Good point, I should probably move this into a loop.